pkg/datapath/iptables/cell.go

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package iptables
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/cidr"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+const (
+	// IPTablesLockTimeout defines the "-w" iptables option when the
+	// iptables CLI is directly invoked from the Cilium agent.
+	IPTablesLockTimeout = "iptables-lock-timeout"
+
+	// DisableIptablesFeederRules specifies which chains will be excluded
+	// when installing the feeder rules
+	DisableIptablesFeederRules = "disable-iptables-feeder-rules"
+
+	// IPTablesRandomFully defines the "--random-fully" iptables option when the
+	// iptables CLI is directly invoked from the Cilium agent.
+	IPTablesRandomFully = "iptables-random-fully"
+)
+
+var Cell = cell.Module(
+	"iptables",
+	"Handle iptables-related configuration for Cilium",
+
+	cell.Config(defaultConfig),
+	cell.Provide(func(
```

Comment:
This is essentially just a layer around the daemonConfig, I don't see the need to expose this as a separate type in hive. I understand it makes sense internally for the purposes of mocking though

Reply:
Yep, main reason to do that is to avoid depending on the big global config object `DaemonConfig`. But it's better to provide it only privately. I'll fix this, thanks :+1: